guides/doc-Provisioning_Guide/images/ProvisioningUserData-Sequence.txt

```diff
@@ -0,0 +1,16 @@
+title VMware Template Provisioning
+participant User
+participant Satellite
+participant VCenter
+participant Capsule X
+
+participant VM
+
+User --> Satellite:Provision VM in network X
+Satellite --> VCenter:Create a VM from template (port 443)
+VCenter --> VM: Clones VM from template
+VM --> Capsule X: Call Back to Sat port 80 via Capsule X 8000
+Capsule X --> Satellite: forward Userdata on 80
+Satellite --> Satellite: redirect to 443
+VM -->Capsule X: Registers itself to Satellite
+Capsule X -->Satellite: proxy registration to satellite
```

Comment:
What's proxy here? Do you refer to Capsule?

Reply:
Capsule proxies registration to satellite. That's what I was referring to.

Reply:
Sorry, I do not get it. I probably lack some context about Capsules. Do you mean smart proxies by chance?

Reply:
Capsule forwards registration to satellite. I think that's probably the more understandable way of saying it.

Reply:
Ah, thank you for clarification. I did not get that "proxy" is a verb here.